dandi/organize.py

```diff
@@ -604,7 +604,6 @@ def is_undefined(d, f):
         age["minimum"] = min(uvs["age"])
         age["maximum"] = max(uvs["age"])
         if age.get("units", None) in (None,) + DEFAULT_VALUES:  # template
-            age.pop("units", None)
```

Comment:
Don't we need to pop off it isn't None?

Reply:
The `insert()` on the next line overwrites any previous value of `age["units"]`.

Reply:
:facepalm: